javagen/src/main/java/com/azure/autorest/mapper/ProxyMethodMapper.java

```diff
@@ -284,9 +287,44 @@ public Map<Request, ProxyMethod> map(Operation operation) {
             }
 
             ProxyMethod proxyMethod = builder.build();
+            proxyMethods.add(proxyMethod);
+
+            ProxyMethodParameter fluxByteBufferParam = parameters.stream()
+                    .filter(parameter -> parameter.getClientType() == GenericType.FluxByteBuffer)
+                    .findFirst()
+                    .orElse(null);
+
+            if (fluxByteBufferParam != null) {
+                List<ProxyMethodParameter> proxyMethodParameters = new ArrayList<>(parameters);
+                int i = parameters.indexOf(fluxByteBufferParam);
+                proxyMethodParameters.remove(i);
+                ProxyMethodParameter binaryDataParam = new ProxyMethodParameter.Builder()
+                        .clientType(ClassType.BinaryData)
+                        .alreadyEncoded(fluxByteBufferParam.getAlreadyEncoded())
+                        .collectionFormat(fluxByteBufferParam.getCollectionFormat())
+                        .defaultValue(fluxByteBufferParam.getDefaultValue())
+                        .description(fluxByteBufferParam.getDescription())
+                        .explode(fluxByteBufferParam.getExplode())
+                        .fromClient(fluxByteBufferParam.getFromClient())
+                        .headerCollectionPrefix(fluxByteBufferParam.getHeaderCollectionPrefix())
+                        .isConstant(fluxByteBufferParam.getIsConstant())
+                        .isNullable(fluxByteBufferParam.getIsNullable())
+                        .name(fluxByteBufferParam.getName())
+                        .isRequired(fluxByteBufferParam.getIsRequired())
+                        .origin(fluxByteBufferParam.getOrigin())
+                        .parameterReference(fluxByteBufferParam.getParameterReference())
+                        .rawType(ClassType.BinaryData)
+                        .requestParameterLocation(fluxByteBufferParam.getRequestParameterLocation())
+                        .requestParameterName(fluxByteBufferParam.getRequestParameterName())
+                        .wireType(ClassType.BinaryData)
+                        .build();
```

Comment:
Recommend to add a `ProxyMethodParameter.newBuilder` in `ProxyMethodParameter`.

When in future, dev add new property to `ProxyMethodParameter`, I guess no one will remember they need to add it here as well.